ras/packages.py

```diff
@@ -27,7 +27,7 @@ def setUp(self):
 
         self.sm = SoftwareManager()
         self.packages = self.params.get(
-            'packages', default=['powerpc-utils', 'ppc64-diag', 'lsvpd'])
+            'packages', default=['powerpc-utils', 'ppc64-diag-rtas', 'lsvpd'])
```

Comment:
@Krishan-Saraswat why are we adding the default packages again here?
your hard coding them in yaml. I feel they are needed again as default packages.
Please move them either to .py file as we have other testcases, or take support of yaml file

Reply:
@Krishan-Saraswat if I understand correctly, ppc64-diag split in two package one is ppc64-diag and other is ppc64-diag-rtas and ppc64-diag is no more default package ?

Reply:
If this correct this will be applicable only for the latest distro then we should still have support for the old distro